docs/zero_to_hero_guide/quickstart.md

```diff
@@ -0,0 +1,185 @@
+# Llama Stack Quickstart Guide
+
+This guide will walk you through setting up an end-to-end workflow with Llama Stack, enabling you to perform text generation using the `Llama3.2-3B-Instruct` model. Follow these steps to get started quickly.
+
+If you're looking for more specific topics like tool calling or agent setup, we have a [Zero to Hero Guide](#next-steps) that covers everything from Tool Calling to Agents in detail. Feel free to skip to the end to explore the advanced topics you're interested in.
+
+## Table of Contents
+1. [Setting up](#Setting-up)
+2. [Build, Configure, and Run Llama Stack](#build-configure-and-run-llama-stack)
+3. [Testing with `curl`](#testing-with-curl)
+4. [Testing with Python](#testing-with-python)
+5. [Next Steps](#next-steps)
+
+---
+
+
+
+## Setting up
+
+### 1. Prerequisite
+
+Ensure you have the following installed on your system:
+
+- **Conda**: A package, dependency, and environment management tool.
+
+
+### 2. Installation
+The `llama` CLI tool helps you manage the Llama Stack toolchain and agent systems.
+
+
+```bash
+pip install llama-stack
+```
+
+After installation, the `llama` command should be available in your PATH.
+
+### 3. Download Llama Models
+
+Download the necessary Llama model checkpoints using the `llama` CLI:
+
+```bash
+llama download --model-id Llama3.2-3B-Instruct
+```
+
+Follow the CLI prompts to complete the download. You may need to accept a license agreement. Obtain an instant license [here](https://www.llama.com/llama-downloads/).
+
+---
+
+## Build, Configure, and Run Llama Stack
+
+### 1. Build the Llama Stack Distribution
+
+We will default to building the `meta-reference-gpu` distribution due to its optimized configuration tailored for inference tasks that utilize local GPU capabilities effectively. If you have limited GPU resources, prefer using a cloud-based instance or plan to run on a CPU, you can explore other distribution options [here](https://llama-stack.readthedocs.io/en/latest/getting_started/index.html#decide-your-inference-provider).
+
+```bash
+llama stack build --template meta-reference-gpu --image-type conda
```

Comment:
I am still seeing this message "You can now run `llama stack configure meta-reference-gpu" @yanxi0830

Reply:
we are getting pypi package updated to fix this, alternatively build by source would fix it